pyproject.toml

```diff
@@ -21,6 +21,8 @@ classifiers = [
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
```

Comment:
If you want to add support to `Python 3.13`, you will need to add it to the test matrix [here](https://github.com/anancarv/python-artifactory/blob/master/.github/workflows/check_code.yml#L52)